resources/views/themes/tailwind/payments/stripe/settings/partials/plans.blade.php

```diff
@@ -0,0 +1,51 @@
+<div class="flex flex-col">
+
+    @subscriber
+    @php
+        $subscriptionName = auth()->user()->getCurrentSubscriptionName();
```

Comment:
could we maybe have some helper for that?

Reply:
Added:
$subscriptionName = current_subscription_name(auth()->user());

if (!function_exists('current_subscription_name')) {
    function current_subscription_name(User $user): ?string
    {
        return $user->getCurrentSubscriptionName();
    }
}